src/audits/performance/first-meaningful-paint.js

```diff
@@ -234,8 +234,9 @@ module.exports = FirstMeaningfulPaint;
 
 /**
  * Math.max, but with NaN values removed
+ * @param {...number} numbers
  */
-function max() {
-  const args = [...arguments].filter(val => !isNaN(val));
+function max(...numbers) {
```

Comment:
we don't have rest parameters in node 5. Can you use the style guide suggestion of `@param {...number} var_args` but then still use `arguments` in the code?